app/api/models.py

```diff
@@ -14,7 +14,7 @@ class QueryModel(BaseModel):
 
     min_age: float = Query(default=None, ge=0)
     max_age: float = Query(default=None, ge=0)
-    sex: Literal["male", "female", "other"] = None
+    sex: Literal["nb:Male", "nb:Female", "nb:OtherSex"] = None
```

Comment:
Referencing [this](https://github.com/neurobagel/project/issues/80#issuecomment-1515275716), i think we decided to go with snomed for sex terms.